full/src/main/java/apoc/mongodb/MongoDB.java

```diff
@@ -164,75 +162,18 @@ public void insert(@Name("host") String hostOrKey, @Name("db") String db, @Name(
                     Map<String, Object> result = coll.first(Map.of(conf.getIdFieldName(), new ObjectId(objectIdValue)));
                     return result == null || result.isEmpty() ? Stream.empty() : Stream.of(new MapResult(result));
                 },
-                e -> log.error("apoc.mongo.get.byObjectId - hostOrKey = [" + hostOrKey + "], db = [" + db + "], collection = [" + collection + "], objectIdValue = [" + objectIdValue + "]",e));
+                e -> log.error("apoc.mongodb.get.byObjectId - hostOrKey = [" + hostOrKey + "], db = [" + db + "], collection = [" + collection + "], objectIdValue = [" + objectIdValue + "]",e));
```

Comment:
Why did this change? Shouldn't it be `apoc.mongo.update` (same name in error as proc we're calling)?

Reply:
This is a modification not strictly related to pr.
I noticed that the procedure is called `apoc.mongodb.get.byObjectId`,
so the error message here was wrong.